### DISCUSSION TITLE
Unidentified Gene Material is incapable of being located via item finder

### DISCUSSION THREAD

**Discussion by Rat-Crusher** [FakeFishGames/Barotrauma#15218] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Title says it all, unidentified gene material is incapable of being located via the item finder. 

### Reproduction steps

1: Use item finder (handheld or ship works)
2: Try to search for Unidentified Gene Material
3: feel despair

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Necronomicron** [comment]
By the way, the bug was there long before the mentioned version. And the item is actually called «Unidentified Genetic Material».
